docs/src/data/meta.ts

```diff
@@ -0,0 +1,253 @@
+export interface MetaInfo {
+  [pathname: string]: { title: string; description: string };
+}
+
+export const META_INFO: MetaInfo = {
+  '/': {
+    title: 'Home',
+    description:
+      'Amplify UI is an open-source design system with cloud-connected components and primitives that simplify building accessible, performant, and beautiful applications in React, Angular, Vue, and Flutter (more coming soon).',
+  },
+  '/getting-started/installation': {
+    title: 'Installation',
+    description:
+      'Installation Guide - How to install Amplify UI Packages, Styles, Fonts and Troubleshooting.',
+  },
+  '/getting-started/migration': {
+    title: 'Migration',
+    description:
+      'Migration Guide - How to migrate from an older version of Amplify UI to a newer version.',
+  },
+  '/components': {
+    title: 'Components',
+    description:
+      'Amplify UI Components, including Connected components, Base components, Feedback components, Navigation components, Input components, Layout components, Utility components.',
+  },
+  '/components/authenticator': {
+    title: 'Authenticator',
+    description:
+      'Authenticator component adds complete authentication flows to your application with minimal boilerplate - Amplify UI',
+  },
+  '/components/chatbot': {
+    title: 'Chatbot',
+    description:
+      'Chatbot is a simple way to add a conversational UI into your app - Amplify UI',
+  },
+  '/components/storage': {
+    title: 'Storage',
+    description:
+      'Storage is a simple way to interact with our storage solutions, such as S3 - Amplify UI',
+  },
+  '/components/divider': {
+    title: 'Divider',
+    description:
+      'Divider creates separations in content, helping to organize content and establish visual rhythm - Amplify UI',
+  },
+  '/components/heading': {
+    title: 'Heading',
```

Comment:
Sorry I didn't notice it before, but can we append Amplify UI at the end of all the titles?
```suggestion
    title: 'Heading - Amplify UI',
```

Reply:
I've done it in _ampp.page.tsx to save some strolls 😁  https://github.com/aws-amplify/amplify-ui/pull/1563/files#diff-71bc81a2d4c2303176a0670d3591211d2ff148b097962a1d0f84e86f79060649L36